rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1238,17 +1238,17 @@ bool game_list_frame::CreatePPUCache(const game_info& game)
 	Emu.SetForceBoot(true);
 	Emu.Stop();
 	Emu.SetForceBoot(true);
-	const bool success = Emu.BootGame(game->info.path, game->info.serial, true);
 
-	if (success)
+	if (const auto error = Emu.BootGame(game->info.path, game->info.serial, true))
 	{
-		game_list_log.warning("Creating PPU Cache for %s", game->info.path);
+		game_list_log.error("Could not create PPU Cache for %s, error: %s", game->info.path, error);
+		return false;
 	}
 	else
```

Comment:
I think you can rmove the else scope and simply return (I personally don't like that the return isn't the last thing in the outer scope)